### PR TITLE
OSDOCS-2113: Release notes for Configuring PROXY protocol for an Ingress Controller

### PR DESCRIPTION
**Pull request by skrthomas**
4.8 release notes for Configuring PROXY protocol for an Ingress Controller: https://issues.redhat.com/browse/OSDOCS-2113

Preview: https://deploy-preview-33241--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-8-networking

@Miciah @quarterpin please take a look at this RN and let me know if you have feedback. Note-this topic is still not merged, so the See link does not work. Merge is pending QA.

### CODE REVIEW

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -163,6 +163,13 @@ For more information, see xref:../networking/ovn_kubernetes_network_provider/tra
 
 For more information, see xref:../networking/ingress-operator.adoc#nw-ingress-converting-http-header-case_configuring-ingress[Converting HTTP header case].
 
+[id="ocp-4-8-ingress-configuring-proxy-protocol"]
+==== Configuring PROXY protocol for an Ingress Controller
+
+{product-title} 4.8 adds support for configuring PROXY protocol for the Ingress Controller API as a way to enhance communication between the Ingress Controller and an external load balancer.
```
Comment (bergerhoffer) [inline release_notes/ocp-4-8-release-notes.adoc:169]: Should references to "PROXY protocol" have a "the" in front ("the PROXY protocol")?

**Review by bergerhoffer — COMMENTED** [review]
Just one question, and then same thing about the link not jumping to the section yet. We should make sure that the main doc PRs are merged before merging the related RNs (so we can check that the links work properly.

**Comment by quarterpin** [comment]
The statement should read as: `OpenShift Container Platform 4.8 adds support for configuring PROXY protocol for the Ingress Controller on non-cloud platforms specifically for "HostNetwork" or "NodePortService" endpoint publishing strategies types`

File: release_notes/ocp-4-8-release-notes.adoc (hunk 1 above)
Comment (skrthomas) [reply release_notes/ocp-4-8-release-notes.adoc:169]: Yep! http://www.haproxy.org/download/2.2/doc/proxy-protocol.txt

File: release_notes/ocp-4-8-release-notes.adoc [hunk 2]
```diff
@@ -200,6 +201,13 @@ For more information, see xref:../networking/ingress-operator.adoc#nw-ingress-co
 {product-title} 4.8 adds support for setting the thread count to increase the amount of incoming connections a cluster can handle.
 
 For more information, see xref:../networking/ingress-operator.adoc#nw-ingress-setting-thread-count_configuring-ingress[Setting Ingress Controller thread count].
+=======
```
Comment (adellape) [inline release_notes/ocp-4-8-release-notes.adoc:204]: Looks like this is a left-over from a merge conflict, which is messing up the rendered version.

**Review by adellape — APPROVED** [review]
:+1:

**Pull request merged**